website/blog/why-we-are-writing-sdks.md

```diff
@@ -0,0 +1,38 @@
+
+<!-- truncate -->
+
+## Introducing Optic SDKs
+Firstly, the idea behind these isn’t to be able to interact with Optics Cloud, or with the learning process of the CLI tool, it’s to simplify capturing the API traffic and sending the captures to the CLI. So when looking at it you won’t see handy API classes etc to interact with.
+
+With that out of the way, let’s take a look at what we have so far, it’s all very work in progress so please pass along your feedback so we can make it just what you want/need sooner rather than later.
+
+Firstly we put together a specification of what should be contained in the SDK, we’ve published this on notion for now to make commenting easier https://www.notion.so/useoptic/Optic-SDK-Specification-ff4d7ba6f0444c9eb0862a6d5748d707.
+What this specification tries to achieve is a language agnostic flow of what the SDK could do, from giivng a point of configuration, through to what it captures, the format is sends in, and where it sends the captured data. 
+
+Currently we have a work in progress Node SDK on GitHub (but not published to NPM) at https://github.com/opticdev/optic-node/tree/main/sdk, and this example just shows a working implementation of our SDK specification.
+## But this isn’t simpler than before?
+Yes reader, you’d be correct about that. It does allow for the more in code configuration of your captures and it does allow for a means to capture your API traffic within your application but it’s not plug and play. But this is step one in our master plan here at Optic to take over the world by being super simple, but that means I little more upfront work.
+Once we have these SDKs it allows for middleware to be produced simply and quickly, as all the hard work has been done up front in the SDK! If you are unaware of what middleware is, here is a one liner:
+
+> Middleware is a type of computer software that provides services to software applications beyond those available from the operating system. It can be described as "software glue".
+> https://en.wikipedia.org/wiki/Middleware
+
+What this means is we want to give you, our amazing users, a simple plug and play module in your desired framework be in Express in Node or Laravel in PHP, we want something you can quickly add to your existing tool chain and everyone working on the project have all the advantages of using Optic. We do have a couple of examples of what the middleware could be like for Express and Hapi in the Node world https://github.com/opticdev/optic-node/tree/main/frameworks, please feel free to comment or log issues that we can take and make it so much better for you.
+## Feel like get involved?
+Optic is only a small organization, we can’t possibly work in every language with every framework so we welcome the communities assistance in doing so. The amazing [Adrien Brault](https://github.com/adrienbrault) has already produced a proof of concept for PHP that you can see at https://github.com/adrienbrault/optic-php.
```

Comment:
worth mentioning bounties?

Reply:
I'd avoid that till we've worked out the kinks of how we want to do it